### ISSUE TITLE
[Bug] Cannot drag folders

### ISSUE THREAD

**Issue by Paalon** [BoostIO/Boostnote#1661]
I cannot drag folders.
My environment:
Boostnote 0.11.1
macOS 10.13.3

**Comment by Paalon** [comment]
I noticed that the draggable area is only on the folder icon. I tried to drag by clicking the text area.

**Comment by rayou** [comment]
yeah, this might a bit confusing. I have suggestion that we can change the mouse style to `cursor: ns-resize;` when user hovers onto the folder icon.

Any thoughts?

**Comment by Paalon** [comment]
That's sounds good.

**Comment by patrick-iv-knowit** [comment]
[Personal opinion] I would prefer just changing the cursor when hovering the folder icon (like suggested in the comment above), having an extra icon for each item just looks weird, especially an icon that (at least on my display) looks like a Pause button.

![capture2](https://user-images.githubusercontent.com/36044123/37388249-0837e7b2-2760-11e8-857c-627a2776b6dc.PNG)

**Comment by Flexo013** [comment]
How about we make thee dragging interface the same between the storages and the folders in the sidebar.

The cursor change in the sidebar uses the native icon, which looks better than the one used in the storages UI. But the (hamburger) icon for dragging is clearer imo than the six dots icon we currently have in the sidebar.

Current storages UI:
![image](https://user-images.githubusercontent.com/4507015/62291644-bc46da00-b464-11e9-94ce-e9a13876f763.png)

**Comment by Flexo013** [comment]
Also on option would be to hide the drag icon until the user hovers over the folder in question.

This what Todoist does:
![image](https://user-images.githubusercontent.com/4507015/62836446-4102d680-bc63-11e9-9b6f-97e916bec4f0.png)
